src/relay_pic/relay_general.c

```diff
@@ -0,0 +1,86 @@
+#include <xc.h>
+#include "relay_general.h"
+
+static uint16_t analog_inputs[2];

```

Comment:
Should this be static? If so, then this variable cannot be used outside of this file

Reply:
Resolved.